sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -110,6 +110,47 @@ void peekMessage() {
             .verifyComplete();
     }
 
+    /**
+     * Verifies that we can schedule and peek a message.
+     */
+    @Test
+    void scheduleMessage() {
+        // Arrange
+        final String messageId = UUID.randomUUID().toString();
+        final String contents = "Some-contents";
+        final ServiceBusMessage message = TestUtils.getServiceBusMessage(contents, messageId, 0);
+        int enqueDelayInSec = 2;
+        // Assert & Act
+        StepVerifier.create(sender.scheduleMessage(message, Instant.now().plusSeconds(enqueDelayInSec))
+            .delaySubscription(Duration.ofSeconds(enqueDelayInSec + 1))
+            .thenMany(receiver.receive().take(1)))
+            .assertNext(receivedMessage -> {
+                Assertions.assertArrayEquals(contents.getBytes(), receivedMessage.getBody());
+                Assertions.assertTrue(receivedMessage.getProperties().containsKey(MESSAGE_TRACKING_ID));
+                Assertions.assertEquals(messageId, receivedMessage.getProperties().get(MESSAGE_TRACKING_ID));
+            })
+            .verifyComplete();
+    }
+
+    /**
+     * Verifies that we can cancel a scheduled message.
+     */
+    @Test
+    void cancelScheduleMessage() {
+        // Arrange
+        final String messageId = UUID.randomUUID().toString();
+        final String contents = "Some-contents";
+        final ServiceBusMessage message = TestUtils.getServiceBusMessage(contents, messageId, 0);
+        int enqueDelayInSec = 3;
+        // Assert & Act
+        StepVerifier.create(sender.scheduleMessage(message, Instant.now().plusSeconds(enqueDelayInSec))
```

Comment:
Scheduling a message isn't part of the scenario. Do that as part of your // Arrange.